src/views/GovernanceTaskList/TaskListCta.tsx

```diff
@@ -109,6 +109,34 @@ export const BusinessCaseDraftCta = ({
           Update draft business case
         </UswdsLink>
       );
+    case 'READY_FOR_GRT':
+      return (
+        <>
+          <Button
```

Comment:
I dont think this needs to be a button. It can be the `UswdsLink` styled as a button.

Reply:
can I style it as a usa-button and a display-block at the same time? I see the other links that are styled as buttons use the className to do it, is there another way?

Reply:
I see that I can do it like this:

className="usa-button display-block margin-bottom-3"

but that causes the button to fill the entire space horizontally, im sure I can fix that with another style option

Reply:
Try `display-inline-block`